Lib/test/test_ntpath.py

```diff
@@ -828,6 +989,10 @@ def test_realpath_permission(self):
             raise unittest.SkipTest('failed to deny access to the test file')
 
         self.assertPathEqual(test_file, ntpath.realpath(test_file_short))
+        with self.assertRaises(OSError):
+            ntpath.realpath(test_file_short, strict=True)
+        with self.assertRaises(OSError):
+            ntpath.realpath(test_file_short, strict='allow_missing')
```

Comment:
Let's remove this test and revisit later?
The `symlink_loop` and `invalid_paths` tests should be enough to cover OSError.
```suggestion
```
My guess is that the GHA user isn't in `S-1-5-32-545` ("A local group that represents all users in the domain.")